src/stan-users-guide/odes.Rmd

```diff
@@ -1,11 +1,41 @@
 # Ordinary Differential Equations  {#ode-solver.chapter}
 
-Stan provides a built-in mechanism for specifying and solving systems
-of ordinary differential equations (ODEs). Three different solvers,
-tuned for solving non-stiff systems and for stiff systems are available.
+Stan provides four different methods for solving systems of ordinary
+differential equations (ODEs). All of these methods will adaptively
+refine their solutions in order to satisfy the given tolerances.
+
+The upside to this is that as long as reasonable tolerances are provided
+and an ODE solver well-suited to the problem is chosen, the choice of timestep
+can be abstracted away. The downside is that it is not always clear from the
+outset what reasonable tolerances are, or which ODE solver is best suited
```

Comment:
choosing "reasonable tolerance" is a universal consideration, not just to adaptive step + tolerance: using fixed stepsize still leads to Q of "how accuracy I want my answer to be?". So I wouldn't say it's a downside of adaptive step & tolerance control.

Reply:
We can instead focus on emphasizing the importance of tolerance, such that user should first think really hard and make decision on the accuracy of the solution _before_ even choosing integrators. The accuracy is tied to their problem, data, as well as expectation. Then they want to investigate the tolerance will give the accuracy they want. This thought process doesn't have to involve we say anything about adaptive stepping.